### ISSUE TITLE
features are extracted successfully but not able to run the video afterwards

### ISSUE THREAD

**Issue by AbhinavJangra29** [ekosman/AnomalyDetectionCVPR2018-Pytorch#215]
(venv) PS D:\ad>  python video_demo.py --feature_extractor "c3d.pickle" --feature_method "c3d" --ad_model "epoch_80000.pt"
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  3.19it/s]
0it [00:00, ?it/s]D:\AnomalyDetectionCVPR2018-Pytorch-main\venv\lib\site-packages\torchvision\io\video.py:161: UserWarning: The pts_unit 'pts' gives wrong results. Please use pts_unit 'sec'.
  warnings.warn("The pts_unit 'pts' gives wrong results. Please use pts_unit 'sec'.")
33it [00:06,  4.91it/s]
D:\ad\video_demo.py:276: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  features = torch.tensor(features)
D:\ad\video_demo.py:140: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  features = torch.tensor(features).to(device)  # pylint: disable=not-callable
DirectShowPlayerService::doRender: Unresolved error code 0x80040266 (IDispatch error #102)

**Comment by ekosman** [comment]
Hi, in order to reproduce the error I need the file that you pass
